app/src/main/java/org/oppia/app/recyclerview/RecyclerDataDiffCallback.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.app.recyclerview
+
+import androidx.recyclerview.widget.DiffUtil
+
+class RecyclerDataDiffCallback<T>(var oldList: List<T>, var newList: List<T>) :
```

Comment:
Could we make the adapter itself be a ``DiffUtil.Callback``?

Reply:
Kotlin doesn't support multiple inheritances that's why I created a separate file, also we can use this `RecyclerDataDiffCallback` as an inner class to our adapter until we have only one adapter in our whole codebase. In future, if we grow to 2 or more adapters we might need `RecyclerDataDiffCallback` for all the adapters.

Reply:
Resolving this on behalf of ben